doc/rtd/explanation/format.rst

```diff
@@ -141,85 +216,150 @@ Create user data containing 3 shell scripts:
 
     $ cloud-init devel make-mime -a always.sh:x-shellscript-per-boot -a instance.sh:x-shellscript-per-instance -a once.sh:x-shellscript-per-once
 
-``include`` file
-================
 
-This content is an :file:`include` file.
+Cloud config archive
```

Comment:
Aside: is cloud config archive nesting supported? I've not checked the code, but I'm curious whether I could stuff have:

- cloud config archive -> cloud config archive -> cloud config archive
- cloud config archive -> mime -> cloud config archive
- mime -> cloud config archive -> mime

and where/when things might start to break with nesting

Reply:
I'll leave that as an exercise for you to figure out :joy: